documentation/data/data-msg-example.md

```diff
@@ -125,34 +125,34 @@ In this example, the Data message contains a value for the DeviceStatus property
 	}
 
 Numeric values must be used for properties defined as `enum` type using `reftypeid`.
-	
+
 ### Type-less Static Data Message
 
 In this example, the Data message extends the 'Tank' type to include a 'Maintenance Schedule' for this instance. The 'Tank' Type itself is not extended.
 
-	[{ 
+	[{
 		"typeid": "Tank",
-		"properties": { 
+		"properties": {
 			"Maintenance Schedule": {
 				"type": "array",
 				"items": {
 					"type": "string",
 					"format": "date-time"
-                		}
+                }
```

Comment:
Minor but we might want to move this closing bracket to proper position.

Reply:
This has been resolved in the newly-added TYPE example.